### PR TITLE
[tests] Add features up to PHP 8

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Good to go :+1:

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "mautic/core-lib": "^5.0"
   },
   "require-dev": {
-    "friendsofphp/php-cs-fixer": "^3.5",
+    "friendsofphp/php-cs-fixer": "^3.14",
```
Comment (escopecz) [inline composer.json:28]: Why was this downgrade needed?

Comment (TomasVotruba) [reply composer.json:28]: Should be upgrade. I made this to trigger the composer update

**Comment by TomasVotruba** [comment]
Rebased and then... ready to go :+1:

**Comment by TomasVotruba** [comment]
Good to go :+1:

**Review by escopecz — APPROVED** [review]
Thanks! 👍

**Pull request merged**